test/core/utils/is-hidden.js

```diff
@@ -40,6 +40,27 @@ describe('axe.utils.isHidden', function() {
 		assert.isFalse(axe.utils.isHidden(document));
 	});
 
+	it('should return false on a `audio` element without `controls` attribute', function() {
+		fixture.innerHTML =
+			'<audio id="target" src="/test/assets/moon-speech.mp3"></audio>';
+		var el = document.getElementById('target');
+		assert.isFalse(axe.utils.isHidden(el));
```

Comment:
This should return `true` as the node is hidden without the `controls` attribute

Reply:
See explanation above - https://github.com/dequelabs/axe-core/pull/1946#discussion_r367033403